dynamic-plugins/imports/package.json

```
@@ -27,9 +27,9 @@
     "@janus-idp/backstage-plugin-quay": "1.11.3",
     "@janus-idp/backstage-plugin-rbac": "1.29.4",
     "@janus-idp/backstage-plugin-tekton": "3.12.3",
-    "@janus-idp/backstage-plugin-bulk-import": "1.4.6",
-    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.1",
-    "@janus-idp/backstage-plugin-topology": "1.27.3",
+    "@janus-idp/backstage-plugin-bulk-import": "1.4.9",
+    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.2",
```

```suggestion
    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.3",
```

A new version of the bulk-import backend plugin has just been released, to fix https://issues.redhat.com/browse/RHIDP-4021 (see https://github.com/janus-idp/backstage-plugins/pull/2178).

https://github.com/janus-idp/backstage-showcase/pull/1635 adds it in the `main` branch.